### DISCUSSION TITLE
About the bug of tool json-to-go-struct

### DISCUSSION THREAD

**Discussion by kN6jq** [he3-app/he3-feedback#110] [Bug Report]
### Version
json-to-go-struct 1.0.1

### Environment
Windows v1.3.1

### Steps to reproduce

![image](https://user-images.githubusercontent.com/27048404/229422360-12277de6-7b0d-4698-b863-32c295286d2e.png)

转出来默认以Data命令

![image](https://user-images.githubusercontent.com/27048404/229422434-8dbacfb0-c572-4ec4-8261-51c594548763.png)

**Comment by lyzhang1999** [comment]
想了解一下，您是希望默认生成唯一的 Struct 名称还是有其他诉求呢？

**Reply by kN6jq** [reply]
默认应该是data  和json里面的重复了   可以起一个其他名字

**Reply by lyzhang1999** [reply]
方便把原 JSON 发一下吗？

**Reply by kN6jq** [reply]
`{"status":0,"msg":"","data":{"total":167,"totalNum":167,"pageCount":17,"page":1,"size":10,"list":[]}}`  大概这个样子

**Reply by lyzhang1999** [reply]
好的，谢谢~
